test/integration/https.spec.ts

```diff
@@ -59,7 +59,11 @@ describe("When configured for HTTPS", () => {
                 https: {
                     keyPath: './test/fixtures/test-ca.key',
                     certPath: './test/fixtures/test-ca.pem',
-                    defaultDomain: 'test.example'
+                    defaultDomain: 'test.example',
+                    countryName: 'UK',
+                    localityName: 'London',
+                    organizationName: 'Test Organiation'
+                    
```

Comment:
We should update the test suite name here - it just says "overriden default domain", but we're actually overriding lots of fields now.